src/pages/consequences.module.scss

```diff
@@ -0,0 +1,56 @@
+@import '../assets/shared-styles/colors.module.scss';
+
+.riskBuying {
+    display: inline-block;
+    font-family: cera-light;
+    font-size:180%;
+    width:300px;
+}
+
+.offenderRegistry {
+    display: inline-block;
+    font-family: cera-bold;
+    font-size: 20px;
+    width: 300px;
+}
+
+.getArrested {
+    display: inline-block;
+    font-family: cera-bold;
+    font-size:20px;
+    padding-top: 50px;
+    padding-bottom: 20px;
+    width: 300px;
+}
+
+.payFines {
+    display: inline-block;
+    font-family: cera-bold;
+    font-size: 20px;
+    padding-top: 50px;
+    padding-bottom: 20px;
+    width: 309px;
+}
+
+.loseJob {
+    display: inline-block;
+    font-family: cera-bold;
+    font-size: 20px;
```

Comment:
Do you need to set the `display`, `font-family`, and `font-size` on each one of these classes? How about just setting them on the p tag so it applies to all. The code principal is called `DRY` code: Don't Repeat Yourself

Reply:
Placed display, font-family, and font-size under a seperate div class to target the paragraph tag.